### PR TITLE
Merge dev into main

### PR DESCRIPTION
**Pull request by AnselemOdims**
# Steps Taken
- Selected and read through the financial modelling API docs
- Built file structures
- Prepared routes and navigation
- Implemented fetching data from the API
- Displayed the data in the home page
- Created the details page
- Wrote unit tests for the application
- Styled the component
- Updated the README file
- Deployed project 

# Presentation
[Presentation Link](https://www.loom.com/share/826b3bcf225b438c9cc1651e917ce8e7)

# Screenshot
demo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,105 @@
...
+<br>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AnselemOdims/Stock-metrics/issues).
+
+<br>
+
```
Comment (elyor-doniyorov) [inline README.md:101]: - [ ] Please add the `Acknowledgements` section that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the author of the original design, as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @AnselemOdims, 

Good job so far!
## To Highlight 👏 
- [ ] There are no linter errors ✔️ 
- [ ] You have a very descriptive title and description 👍 
- [ ] You have a great video presentation 💯 
- [ ] Your app looks awesome 🥇 

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

```

File: README.md (hunk 1 above)
Comment (AnselemOdims) [reply README.md:101]: Ok thanks

**Comment by enniopo** [comment]
Hi @AnselemOdims, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AnselemOdims** [comment]
Thanks for the review @enniopo

**Pull request merged**